test/api_test.py

```diff
@@ -80,18 +68,14 @@ def test_peer_add(gd2client):
 
 
 def test_peer_status(gd2client):
-    """
-    This function tests for peer status.
-    """
+    """Test for peer status."""
     status, _ = gd2client.peer_status()
     # assert resp[0]['id'] == Peer.get_peer_id)
     assert status == 200
 
 
 def test_peer_remove(gd2client):
-    """
-    This function tests for peer remove.
-    """
+    """Tests for peer remove."""
```

Comment:
Nit: Not sure if this is grammatically correct. Should it be: Test peer removal. Or Test for peer removal. 

I'm actually not sure, I may be wrong.

Reply:
I think I had implied, "Tests for peer remove operation" but "Test peer removal" is more precise.
@sac Thanks.